docs/samples/v1beta1/torchserve/v1/bloom/bloom-560m.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  name: "torchserve-bloom-560m"
+spec:
+  predictor:
+    pytorch:
+      storageUri: pvc://model-cache
+      resources:
+          limits:
+            cpu: 4
+            memory: 16Gi
+            nvidia.com/gpu: 1
+  nodeName: ip-xxx-xxx-xxx-xxx.us-west-2.compute.internal
```

Comment:
Do we need this `nodeName` field?

Reply:
Yes, since we are provisioning storage from local disk. The isvc should be deployed in the node that has the volume.